README.md

```diff
@@ -1,6 +1,7 @@
 # go-paymail
 > Paymail client & server library for Golang
 
+[SPV Wallet GitBook](https://bsvblockchain.gitbook.io/docs)
```

Comment:
IMHO this link should be introduced with some text, e.g. what it is and what an user can find there.